lib/collection.js

```diff
@@ -382,9 +382,11 @@ Collection.prototype.find = function(query, options, callback) {
   // Translate to new command option noCursorTimeout
   if (typeof newOptions.timeout === 'boolean') newOptions.noCursorTimeout = newOptions.timeout;
 
+  const excludedOptions = new Set(['session', 'collation']);
+
   // Merge in options to command
   for (let name in newOptions) {
```

Comment:
I feel like we have a utility method for this, and if not we should. I believe its called `decorateCommand`, and it should take an array/set of options to exclude. Can we switch to using some sort of utility function here?